l10n_nl_xaf_auditfile_export/__openerp__.py

```diff
@@ -13,6 +13,9 @@
         'base',
         'account',
     ],
+    "demo": [
+        "demo/res_partner.xml"
+    ],
     "data": [
         "demo/res_partner.xml",
```

Comment:
I think you forgot to remove this line inside "data" ?

Reply:
Very true, fixed now.